debs/generate_debian_package.sh

```diff
@@ -86,8 +91,11 @@ build() {
         elif [[ "${ARCHITECTURE}" = "i386" ]]; then
             BUILD_NAME="${DEB_I386_BUILDER}"
             FILE_PATH="${DEB_I386_BUILDER_DOCKERFILE}"
+	elif [[  "${ARCHITECTURE}" = "ppc64le" ]]; then
```

Comment:
Little indentation issue here, please review

Reply:
I have tried to resolve the indentation issue, please check..